src/modules/previewpane/MarkdownPreviewHandler/MarkdownTelemetry.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+using PreviewHandlerCommon.Telemetry;
+
+namespace MarkdownPreviewHandler
+{
+    /// <summary>
+    /// Telemetry helper class for markdown renderer.
+    /// </summary>
+    public class MarkdownTelemetry : TelemetryBase
+    {
+        /// <summary>
+        /// Name for ETW event.
+        /// </summary>
+        private const string EventSourceName = "Microsoft.PowerToys";

```

Comment:
nit: maybe this could be in the base :) ?

Reply:
It's possible that same projects can have multiple event sources. That's why it's better to initialize them from the derived classes.